app/include/dt-bindings/zmk/rgb.h

```diff
@@ -17,7 +17,8 @@
 #define RGB_SPD_CMD 10
 #define RGB_EFF_CMD 11
 #define RGB_EFR_CMD 12
-#define RGB_COLOR_HSB_CMD 13
+#define RGB_EFS_CMD 13
```

Comment:
Is this required to get each side synced?

Reply:
You saw why elsewhere, marking resolved! Thanks for the review!